src/CommunityToolkit.Maui.Core/Interfaces/IDrawingView.shared.cs

```diff
@@ -53,9 +53,27 @@ public interface IDrawingView : IView
 	/// </summary>
 	void Clear();
 
+	/// <summary>
+	/// Event occurred when drawing line started
+	/// </summary>
+	/// <param name="point">Last drawing point</param>
+	void OnDrawingLineStarted(PointF point);
+	
+	/// <summary>
+	/// Event occurred when drawing line cancelled
+	/// </summary>
+	void OnDrawingLineCancelled();
+	
+	/// <summary>
+	/// Event occurred when point drawn
+	/// </summary>
+	/// <param name="point">Last drawing point</param>
+	void OnPointDrawn(PointF point);
+	
 	/// <summary>
 	/// Event occurred when drawing line completed
 	/// </summary>
 	/// <param name="lastDrawingLine">Last drawing line</param>
+	[Obsolete("This will be renamed to OnDrawingLineCompleted in a future version of this library.")]
```

Comment:
```suggestion
	[EditorBrowsable(EditorBrowsableState.Never)]
	[Obsolete("This will be renamed to OnDrawingLineCompleted in a future version of this library.")]
```

Reply:
As we just rename it without providing an alternative I suppose it should be visible in editor

Reply:
Instead of teeling that will be renamed, what not create the new name and guide devs to use it and delete the old name in the next release?

Reply:
rename is simpler for us, rather than adding a lot of code in each layer (interface, handler, etc)